### PR TITLE
Adds lecture list to comments section of portfolio

### PR DESCRIPTION
**Pull request by jaysaleh**
This pull request utilizing the `doGet` method implemented previously to generate a list of lectures.

Screenshots: https://screenshot.googleplex.com/jqMmZVTzpaM.png

### CODE REVIEW

File: src/main/webapp/index.html [hunk 1]
```diff
@@ -3,17 +3,18 @@
...
-  <h2>Add a new lecture!<h2>
+  <h2>Add a new lecture!</h2>
   <h3>Enter a name for this lecture:</h3>
-  <input type="text" placeholder="Lecture Name" name="name-input">
-  <br/>
-  <h3>Enter the YouTube link for this lecture:<h3>
-  <input type="text" placeholder="Name" name="video-input">
-  <br/>
-  <br/>
+  <input type="text" placeholder="Lecture Name" name="name-input" />
```
Comment (amuamushu) [inline src/main/webapp/index.html:9]: You shouldn't have a backslash for br and input 
Source: https://google.github.io/styleguide/htmlcssguide.html#Document_Type

File: src/main/java/com/googleinterns/zoomtube/servlets/LectureServlet.java [hunk 2]
```diff
@@ -14,16 +14,16 @@
...
 import com.google.appengine.api.datastore.DatastoreService;
 import com.google.appengine.api.datastore.DatastoreServiceFactory;
 import com.google.appengine.api.datastore.Entity;
 import com.google.appengine.api.datastore.PreparedQuery;
 import com.google.appengine.api.datastore.Query;
+import com.google.gson.Gson;
+import com.google.sps.data.Lecture;
 import java.io.IOException;
+import java.util.*;
+import java.util.regex.*;
```
Comment (n-wach) [inline src/main/java/com/googleinterns/zoomtube/servlets/LectureServlet.java:26]: Wildcard imports are not allowed by the styleguide: https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports

File: src/main/webapp/index.html [hunk 3]
```diff
@@ -3,17 +3,18 @@
...
+  <input type="text" placeholder="Name" name="video-input" />
+  <br />
+  <br />
   <div id="submit-button">
-    <input type="submit"/>
+    <input type="submit" />
   </div>
 </form>
+<ul id="lecture-list"></ul>
+<script src="script.js"></script>
```
Comment (n-wach) [inline src/main/webapp/index.html:20]: An alternative to bringing script to the end of the doc is adding the `defer` tag, which waits until the entire index.html is ready before running.  IDK which is preferable
```suggestion
<script src="script.js" defer></script>
```

File: src/main/webapp/index.html (hunk 1 above)
Comment (jaysaleh) [reply src/main/webapp/index.html:9]: `make pretty` adds those

File: src/main/java/com/googleinterns/zoomtube/servlets/LectureServlet.java (hunk 2 above)
Comment (jaysaleh) [reply src/main/java/com/googleinterns/zoomtube/servlets/LectureServlet.java:26]: fixed

File: src/main/webapp/index.html (hunk 3 above)
Comment (jaysaleh) [reply src/main/webapp/index.html:20]: fixed

File: src/main/webapp/index.html [hunk 4]
```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE html>
 <meta charset="utf-8" />
-
+<script src="script.js" defer></script>
 <title>ZoomTube</title>
 <link rel="stylesheet" href="style.css" />
 <script src="script.js"></script>
```
Comment (n-wach) [inline src/main/webapp/index.html:6]: Script already included above.  Maybe make this the deferred one and delete the other? kinda weird having it before `title`

File: src/main/webapp/script.js [hunk 5]
```diff
@@ -0,0 +1,33 @@
+loadLectureList();
```
Comment (n-wach) [inline src/main/webapp/script.js:1]: Does this work? Can you call the function before it's defined?

File: src/main/webapp/index.html (hunk 4 above)
Comment (jaysaleh) [reply src/main/webapp/index.html:6]: missed it, thanks

File: src/main/webapp/script.js (hunk 5 above)
Comment (jaysaleh) [reply src/main/webapp/script.js:1]: Yeah it works by hoisting, https://scotch.io/tutorials/understanding-hoisting-in-javascript#:~:text=Hoisting%20is%20a%20JavaScript%20mechanism,scope%20is%20global%20or%20local.

Comment (n-wach) [reply src/main/webapp/script.js:1]: Didn't know that, cool!

**Pull request merged**